src/dkg/complaint.rs

```diff
@@ -29,6 +30,8 @@ pub struct Complaint<C: CipherSuite> {
     pub proof: ComplaintProof<C>,
 }
 
+impl_serialization_traits!(Complaint<CipherSuite>);
```

Comment:
Note that wherever I am calling the macro, I use the full `Ciphersuite` name even though the trait bound is enforced inside the macro, for clarity. But it could be changed to something like `Complaint<C>` or else, if judged more appropriate.